manifests/vhost.pp

```diff
@@ -338,23 +338,24 @@
   }
 
   if $ip {
+    $_ip = any2array($ip)
     if $port {
-      $listen_addr_port = "${ip}:${port}"
-      $nvh_addr_port = "${ip}:${port}"
+      $listen_addr_port = parseyaml(inline_template('<% res="" %><% @_ip.each { |a_ip| res = res + "#{a_ip}:#{@port}," } %><%= res.split(",").to_yaml -%>'))
```

Comment:
Let's call this an interesting approach.. @igalic : Would you suggest using a custom function for this? Nevertheless, this could be done a lot easier with future_parser enabled...